Examples/Examples/FormViewExampleView.swift

```diff
@@ -67,6 +74,15 @@ struct FormViewExampleView: View {
                         .padding([.horizontal])
                 }
             
+                .alert("Cancel editing?", isPresented: $cancelConfirmationIsShowing) {
+                    Button("Cancel editing", role: .destructive) {
```

Comment:
The AGOL map viewer has this:

![image](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/3998072/4e82a0e7-be05-4ce5-896b-a695cfadce26)

So instead of "Cancel editing", maybe "Discard edits" as that is more descriptive. We should coordinate with Kotlin as well to make sure we're in sync with the verbiage.

Reply:
Per discussion in Apollo Kotlin currently has no interface for this and just dismisses the form.